chrome/content/zotero/overlay.js

```diff
@@ -188,7 +189,7 @@ var ZoteroOverlay = new function()
 		Zotero.debug("zoteroPane.getAttribute('minheight'): " + zoteroPane.getAttribute('minheight'));
 		Zotero.debug("savedHeight: " + savedHeight);
 		*/
-		
+
```

Comment:
Once again, my IDE does removes trailing whitespace automatically for the lines I touch. It is usually a good thing too: less whitespace means smaller files. I was looking into turning it off, but decided not to, since it probably does more good than bad. Opinions?